src/routes/api/dashboard/test-attempts/server.spec.ts

```
@@ -0,0 +1,79 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest';
+import { GET } from './+server';
+import { BACKEND_URL } from '$env/static/private';
+
+vi.mock('$lib/server/auth', () => ({
+	getSessionTokenCookie: vi.fn(() => 'mock-token')
+}));
+
+const mockFetch = vi.fn();
+global.fetch = mockFetch as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)